src/dug/core/async_search.py

```diff
@@ -621,17 +349,18 @@ async def search_vars_unscored(self, concept="", query="",
                 "name": elem_s['element_name']
             }
 
+            if scored:
+                elem_info["score"]: round(elem['_score'], 6)
```

Comment:
Do we really need scores to 6 digits of precision? I would think 3 or 4 should be enough, but I don't have a strong opinion on this. Might be worth discussing in another ticket.

Reply:
I guess you are right, but i did not author this line of code, just moved it.